test/dbus/lvmdbustest.py

```diff
@@ -1865,6 +1867,46 @@ def test_nesting(self):
 		for i in range(0, 5):
 			pv_object_path = self._create_nested(pv_object_path)
 
+	def test_pv_symlinks(self):
+		# Lets take one of our test PVs, pvremove it, find a symlink to it
+		# and re-create using the symlink to ensure we return an object
+		# path to it.  Additionally, we will take the symlink and do a lookup
+		# (Manager.LookUpByLvmId) using it and the original device path to
+		# ensure that we can find the PV.
+		symlink = ""
+
+		pv = self.objs[PV_INT][0]
+		pv_device_path = pv.Pv.Name
+
+		self._pv_remove(pv)
+
+		# Make sure we no longer find the pv
+		rc = self._lookup(pv_device_path)
+		self.assertEqual(rc, '/')
+
+		# Lets locate a symlink for it
+		devices = glob('/dev/disk/*/*')
+		for d in devices:
+			if pv_device_path == os.path.realpath(d):
+				symlink = d
+				break
+		else:
+			self.assertTrue(False, "We expected to find at least 1 symlink!")
```

Comment:
``self.assertIsNotNone(symlink, ...)`` after the ``for`` cycle would be better here. And the ``for`` cycle could be replaced with a ``next()`` call using a generator expression and ``None`` as the default value. But I can imagine the ``for`` cycle can be better readable.